github/github.go

```diff
@@ -75,3 +76,25 @@ func UnblockUser(org string, user string) (bool, error) {
 
 	return true, nil
 }
+
+// Locks all of a user's issues and PRs in the given repository
+//
+// Returns all successfully locked issues/PRs
+func LockAll(org string, repository string, user string, reason string) ([]int64, error) {
+	// All pull requests are issues, but not all issues are pull requests
+	issues, _, err := client.Issues.ListByRepo(context.Background(), org, repository, &github.IssueListByRepoOptions{Creator: user, State: "all"})
+	if err != nil {
+		return nil, err
+	}
+
+	lockedIssues := make([]int64, 0, len(issues)) // Pre-allocate a slice with capacity
+	for _, issue := range issues {
+		_, err := client.Issues.Lock(context.Background(), org, *issue.Repository.Name, *issue.Number, &github.LockIssueOptions{LockReason: reason})
```

Comment:
I think the most important thing is to close the issue, I would make this command be a lock and close.

Reply:
I had assumed locking an issue also closes it, if that's not the case I'll do it explicitly

Reply:
it doesn't.